hazelcast-jet-spring/pom.xml

```diff
@@ -131,5 +131,10 @@
             <version>${spring.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.3.2</version>
+        </dependency>
```

Comment:
I believe this is only needed for test scope, since `hazelcast-jet-spring` module itself doesn't require this dependency?

Reply:
If the annotations go unnoticed in test scope without this dep, why would't they at runtime?

Reply:
You’re not using any annotation in hazelcast-jet-spring module, how could it cause any problem at runtime?